utxo/src/utxo_impl/simulation.rs

```diff
@@ -0,0 +1,147 @@
+//TODO: need a better way than this.
+
+use crate::utxo_impl::test_helper::{create_utxo, DIRTY, FRESH};
+use crate::{flush_to_base, UtxosCache, UtxosView};
+use crate::{UtxoEntry, UtxoStatus};
+use common::chain::OutPoint;
+use common::primitives::{Id, H256};
+use crypto::random::{make_pseudo_rng, Rng};
+
+fn random_bool() -> bool {
+    let rng = make_pseudo_rng().gen_range(0..2);
+    rng == 0
+}
```

Comment:
`make_pseudo_rng().gen::<bool>()`?

Reply:
Welcome to translating things literally from C++ xD

Fixed.